sw/airborne/modules/computer_vision/snake_gate_detection.c

```diff
@@ -147,7 +148,7 @@ int snake_gate_detection(struct image_t *img, int n_samples, int min_px_size, fl
 
   int x, y;
   best_quality = 0;
-  best_gate.quality = 0;
+  (*best_gate).quality = 0;
```

Comment:
I think `best_gate->quality` is cleaner but it's not that important

Reply:
For a next time : )

Reply:
hint:
```
sed -i 's#(*best_gate).#best_gate->#g' snake_gate_detection.c
```

Reply:
Done